### PR TITLE
Update README for main and iOS sections.

### PR DESCRIPTION
**Pull request by Mehequanna**
This is just the first of many updates to the Readme files.

### CODE REVIEW

File: iOS/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Setup instructions for AugmentedFacesExample app.
+
+From the /MMIWG2S/iOS directory in terminal:
+1. Run `$ pod install`
```
Comment (callum-code) [inline iOS/README.md:4]: Could be good to include some instructions for installing cocoapods.

Comment (Mehequanna) [reply iOS/README.md:4]: Thanks! I forget about that since I've had it installed since the beginning of time haha.

**Comment by mamaro556** [comment]
Yeah, I was going to say include the signings and capabilities problem.  It looks good to me.

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-### MMIW
+# MMIW AR Face Mesh App
+
+## Project
+Our goal is to bring awareness to MMIW through the use of augmented reality face
+ meshing technology. Users will learn about MMIW and then be able to take a picture
+ with the iconic red hand over their face. They will then be able to share that
+ picture with whatever social media or device they choose.
```
Comment (Mehequanna) [inline README.md:7]: Fix the spacing here.

**Pull request merged**